src/components/SdkSwitcher/index.tsx

```diff
@@ -0,0 +1,135 @@
+import clsx from 'clsx';
+import React, { Fragment, useState } from 'react';
+import { useHistory } from 'react-router';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+import { Listbox, Transition } from '@headlessui/react';
+import { CheckIcon, SelectorIcon } from '@heroicons/react/solid';
+import styles from './styles.module.css';
+import { sdks } from '@site/src/constants';
+
+interface SdkItem {
+  id: number;
+  name: string;
+  url: string;
+}
+interface SdkList extends Array<SdkItem> {}
+
+const SDK_ID_KEY = 'imx-docs-core-sdk-id';
+
+const classNames = (...classes: string[]): string => {
+  return classes.filter(Boolean).join(' ');
+};
+
+const getSdkId = (sdkList: SdkList): number => {
+  const localId = parseInt(localStorage.getItem(SDK_ID_KEY)) || 0;
+
+  if (localId > sdkList.length - 1) {
+    return 0;
+  }
+  return localId;
+};
+
+const SdkSwitcher = () => {
+  const history = useHistory();
+  const sdkId = getSdkId(sdks);
+  const [selectedSdk, setSelectedSdk] = useState(sdks[sdkId]);
+
+  const onSelect = (value) => {
```

Comment:
I prefer to prefix binding methods with `handle`, it's a little less ambiguous.

Since this is for an `onChange` event, it would be good to make this `onChange={handleOnChange}`

```suggestion
  const handleOnChange = (value) => {
```

If you could also type `value` param as similarly suggested above, that would be great too :)